channels/ps.m3u

```diff
@@ -29,13 +23,11 @@ http://htvpalsat.mada.ps:8888/PBC/tracks-v1a1/mono.m3u8
 https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/palestinian_satellite_channel/hls1/palestinian_satellite_channel.m3u8
 #EXTINF:-1 tvg-id="PalestineToday.ps" tvg-name="Palestine Today" tvg-country="PS" tvg-language="Arabic" tvg-logo="https://paltoday.tv/style/assets/images/logo.png" group-title="News",Palestine Today (480p)
 https://live.paltoday.tv/paltv/live/playlist.m3u8
-#EXTINF:-1 tvg-id="PALESTINIANTV.ps" tvg-name="PALESTINIAN TV" tvg-country="PS" tvg-language="" tvg-logo="" group-title="",PALESTINIAN TV
-http://51.255.84.28:8081/palestiniantv_source/live/chunks.m3u8
-#EXTINF:-1 tvg-id="palestiniantv.ps" tvg-name="palestiniantv" tvg-country="PS" tvg-language="" tvg-logo="" group-title="",palestiniantv
-http://palestiniantv.origin.technostreaming.net:8081/palestiniantv_source/live/chunks.m3u8
-#EXTINF:-1 tvg-id="WatarTV.ps" tvg-name="Watar TV" tvg-country="PS" tvg-language="" tvg-logo="" group-title="",Watar TV
-http://htvint.mada.ps:8889/orient/tracks-v1a1/mono.m3u8?v=86757236
-#EXTINF:-1 tvg-id="WattanTV.ps" tvg-name="Wattan TV" tvg-country="PS" tvg-language="" tvg-logo="" group-title="",Wattan TV (720p)
-http://htvmada.mada.ps:8888/wattan/index.m3u8?v=86757236
-#EXTINF:-1 tvg-id="WattanTV.ps" tvg-name="Wattan TV" tvg-country="PS" tvg-language="" tvg-logo="" group-title="",Wattan TV
-http://htvmada.mada.ps:8888/wattan/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="PALESTINIANTV.ps" tvg-name="PALESTINIAN TV" tvg-country="PS" tvg-language="Arabic" tvg-logo="" group-title="",PALESTINIAN TV
+http://51.255.84.28:8081/palestiniantv_source/live/playlist.m3u8
+#EXTINF:-1 tvg-id="palestiniantv.ps" tvg-name="palestiniantv" tvg-country="PS" tvg-language="Arabic" tvg-logo="" group-title="",palestiniantv
+http://palestiniantv.origin.technostreaming.net:8081/palestiniantv_source/live/playlist.m3u8
+#EXTINF:-1 tvg-id="WatarTV.ps" tvg-name="Watar TV" tvg-country="PS" tvg-language="Arabic" tvg-logo="" group-title="",Watar TV
+http://htvint.mada.ps:8889/orient/index.m3u8
```

Comment:
Watar TV and Orient Radio + has got the same link.

+ I don't seem to have any image... is it a FM radio ?